powertools-core/src/main/java/software/amazon/lambda/powertools/core/internal/LambdaHandlerProcessor.java

```diff
@@ -56,19 +56,19 @@ public static boolean placedOnStreamHandler(final ProceedingJoinPoint pjp) {
                 && pjp.getArgs()[2] instanceof Context;
     }
 
-    public static Optional<Context> extractContext(final ProceedingJoinPoint pjp) {
+    public static Context extractContext(final ProceedingJoinPoint pjp) {
```

Comment:
Avoiding too many usage of Optional

Reply:
is this a problem?

Reply:
Yeah not generally .. but i am trying to avoid any extra computations .. Intresting blog which i referred before when working on it https://struberg.wordpress.com/2017/01/28/optional-vs-if-null/..